### PR TITLE
Query : Get Instances

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for `GetInstances`

## Changes
Added support for `GetInstances`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance
By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

Depends on #85

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,12 @@ SemVer getVersion()
 ```
 Retrives the node software version.
 
+- `getInstances`
+```java
+ImmutableList<ContractAddress> getInstances(Hash blockHash)
+```
+Get the list of smart contract instances in a given block.
```
Comment (EmilLa1) [inline README.md:222]:
```suggestion
Get a list of all smart contract instances that exist in the state at the end of the given block.
```

File: README.md [hunk 2]
```diff
@@ -373,6 +379,12 @@ val peers = client.getPeerList(true);
 SemVer version = client.getVersion();
 ```
 
+#### getInstances
+```java
+val contractInstances = client
```
Comment (EmilLa1) [inline README.md:384]: Please provide type.

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 3]
```diff
@@ -313,6 +316,22 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the list of smart contract instances in a given block.
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/Client.java:320]: This is not entirely correct. 
The list returned will contain all instances that exists at the end of the given block. 

So something like "Get a list of all smart contract instances that exist in the state at the end of the given block." will better describe it.

**Review by EmilLa1 — CHANGES_REQUESTED** [review]
Looks good.
Please provide a unit test.

File: README.md (hunk 2 above)
Comment (parv0888) [reply README.md:384]: done

**Review by EmilLa1 — APPROVED** [review]
Looks good

**Pull request merged**